frontend/packages/dev-console/src/components/topology2/TopologyPage.tsx

```diff
@@ -86,16 +92,31 @@ const TopologyPage: React.FC<Props> = ({
         variant={showListView ? NamespacedPageVariants.light : NamespacedPageVariants.default}
         hideApplications={showListView}
         toolbar={
-          <Tooltip position="left" content={showListView ? 'Topology View' : 'List View'}>
-            <Link
-              className="pf-c-button pf-m-plain"
-              to={`/topology/${namespace ? `ns/${namespace}` : 'all-namespaces'}${
-                showListView ? '' : '/list'
-              }`}
-            >
-              {showListView ? <TopologyIcon size="md" /> : <ListIcon size="md" />}
-            </Link>
-          </Tooltip>
+          <div style={{ display: 'flex' }}>
+            {!showListView && !allProjectsView && (
+              <Popover
+                aria-label="Shortcuts"
+                bodyContent={TopologyShortcuts}
+                position="left"
+                maxWidth="50rem"
+              >
+                <Button type="button" variant="link" className="odc-topology__shortcuts-button">
```

Comment:
Shouldn't need the extra classes on the icon. Move it to the button:
```suggestion
                <Button type="button" variant="link" className="odc-topology__shortcuts-button" icon={<QuestionCircleIcon />}>
```
Looks the same:
![image](https://user-images.githubusercontent.com/14068621/69279488-739d6300-0bb2-11ea-852f-47877f5313e2.png)